src/modules/iframe/components/iframeGenerator/IFrameGenerator.js

```diff
@@ -0,0 +1,209 @@
+import { html } from 'lit-html';
+import { $injector } from '../../../../injection';
+import { emitNotification, LevelTypes } from '../../../../store/notifications/notifications.action';
+import { MvuElement } from '../../../MvuElement';
+import { PathParameters } from '../../../../domain/pathParameters';
+import clipboardIcon from './assets/clipboard.svg';
+import css from './iframegenerator.css';
+import { IFRAME_ENCODED_STATE } from '../../../../utils/markup';
+
+const Update_Size_Width = 'update_size_width';
+const Update_Size_Height = 'update_size_height';
+const Update_Auto_Width = 'update_auto_width';
+const Update_Preview_Url = 'update_preview_url';
+
+const Auto_Width = '100%';
+const Range_Min = 100;
+const Range_Max = 2000;
+
+/**
+ * Component to preview the embedded version
+ *
+ * @author thiloSchlemmer
+ * @author alsturm
+ */
+export class IFrameGenerator extends MvuElement {
```

Comment:
We should think about a coherent notation: `IFrame...` or `Iframe...` (see `IframeStatePlugin`)

Reply:
ok, then let's stick with `Iframe...`